contracts/core/modules/lib/RebalancingSetIssuance.sol

```diff
@@ -0,0 +1,203 @@
+/*
+    Copyright 2019 Set Labs Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+pragma solidity 0.5.7;
+
+import { SafeMath } from "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+import { ERC20Wrapper } from "../../../lib/ERC20Wrapper.sol";
+import { IRebalancingSetToken } from "../../interfaces/IRebalancingSetToken.sol";
+import { ISetToken } from "../../interfaces/ISetToken.sol";
+import { ModuleCoreState } from "./ModuleCoreState.sol";
+
+
+/**
+ * @title RebalancingSetIssuance
+ * @author Set Protocol
+ *
+ * The RebalancingSetIssuance contains utility functions used in RebalancingSet 
+ * issuance
+ */
+contract RebalancingSetIssuance is 
```

Comment:
Where are tests for these? Prefer they be tested separately instead of by proxy through the module. Will allow you to reuse them with confidence.

Also, you won't leave the person who changes the `RebalancingSetIssuanceModule` contract holding the bag when the module contract changes and one of these is no longer used.